src/fragments/lib/auth/js/manageusers.mdx

```diff
@@ -131,7 +131,14 @@ Auth.currentSession()
 
 ## Managing user attributes
 
-You can pass user attributes during sign up:
+Attributes such as name, email address, phone number help you identify individual users. Amplify JS library contains 
+a set of APIs that can help you manage attributes in your user pool.
+
+### Pass user attributes during sign up
+
+Amazon Cognito has a set of default standard attributes as well as an availability to add custom attributes.
```

Comment:
Amazon Cognito has a set of default standard attributes, you have the ability to define add custom attributes.